marimo/_pyodide/pyodide_session.py

```diff
@@ -321,9 +315,9 @@ def launch_pyodide_kernel(
     )
 
     if is_edit_mode:
-        from marimo._output.formatters.formatters import register_formatters
-
-        register_formatters()
```

Comment:
was this just not needed before?

Reply:
I'm going to add this back, for both edit/run mode